src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,392 @@
+    reliabilities = np.random.rand(num_iteration)    # get K random reliability values from an
+                                                     # uniform distribution over [0,1)
+    reliabilities = np.append(reliabilities, [0.5])  # add 0.5 (for median loss) as
+                                            # a last value to reliabilities array
+    results = wf_itm.CalcItmPropagationLoss(lat_cbsd, lon_cbsd, h_cbsd, lat_c, lon_c, h_c,
+                                            indoor_cbsd,
+                                            reliability=reliabilities,
+                                            freq_mhz=FREQ_PROP_MODEL)
+    path_loss = np.array(results.db_loss)
+
+    # Compute CBSD EIRP inside the frequency range of protection constraint
+    eirpPerMHz_cbsd = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                      ant_az_cbsd, ant_beamwidth_cbsd,
+                                                      maxEirpPerMHz_cbsd)
+    eff_bandwidth = min(high_freq_cbsd, high_freq_c) - max(low_freq_cbsd, low_freq_c)
+    eirp_cbsd = eirpPerMHz_cbsd + 10 * np.log10(eff_bandwidth / 1000000.0)
+
+    # Calculate the interference contributions
+    interf = eirp_cbsd - path_loss
+    median_interf = interf[-1]      # last element is the median interference
+    K_interf = interf[:-1]          # first 'K' interference
+
+    # Store interference contributions
+    interference = {
+        'grantIndex': grant_index,
+        'medianInterference': median_interf,
+        'randomInterference': K_interf,
+        'bearing_c_cbsd': results.incidence_angles.hor_rx}
+
+    return interference
+
+
+def formInterferenceMatrix(cbsd_grant_list, constraint, num_iter):
+
+    """
+    Form the matrix of interference contributions to protection constraint c.
+
+    Inputs:
+        cbsd_grant_list:    a list of CBSD grants
+        constraint:         protection constraint, a tuple with named fields of
+                            'latitude', 'longitude', 'low_frequency', 'high_frequency'
+        num_iter:           number of random iterations
+
+    Returns:
+        I:                  a matrix of interference contributions to protection
+                            constraint (c) with dimensions (num_iter x len(cbsd_grant_list))
+        sorted_grant_index: a list of indices of the sorted grant list
+        sorted_bearing:     a list of bearings from protection point to CBSDs of the
+                            sorted grant list
+    """
+
+    # Compute interference contributions of each grant to the protection constraint
+    interf_list = []
+    for cbsd_grant in cbsd_grant_list:
+        interf = computeInterference(cbsd_grant, constraint, num_iter)
+        interf_list.append(interf)
+
+    # Sort grants by their median interference contribution, smallest to largest
+    interf_list.sort(key=itemgetter('medianInterference'))
```

Comment:
In case you decide to chang interf to namedtuple (see previous comment), then you can get easily the sorting with:
```
 from operator import attrgetter
 interf_list.sort(key=attrgetter('medianInterference'))
 sorted_grant_index = [interf.grantIndex for interf in interf_list]
...
```

Reply:
Got it. Thanks!